tests/MSBuildDeviceIntegration/Tests/DebuggingTest.cs

```diff
@@ -248,7 +236,6 @@ public override void OnCreate ()
 					EvaluationOptions = EvaluationOptions.DefaultOptions,
 				};
 				options.EvaluationOptions.UseExternalTypeResolver = true;
-				ClearAdbLogcat ();
 				b.BuildLogFile = "run.log";
 				Assert.True (b.RunTarget (proj, "_Run", doNotCleanupOnUpdate: true, parameters: new string [] {
```

Comment:
Should this be refactored to use the new `RunProjectAndAssert` ?
I know it is setting the debug port as well, but `RunProjectAndAssert` might be able to do both.

Reply:
Fixed.